packages/jest-mock/__typetests__/mock-functions.test.ts

```diff
@@ -135,6 +136,9 @@ if (returnValue.type === 'throw') {
   expectType<unknown>(returnValue.value);
 }
 
+const context = mockFn.mock.contexts[0];
+expectType<Date>(context);
```

Comment:
Perhaps slightly less code? Not important at all (;

```suggestion
expectType<Array<Date>>(mockFn.mock.contexts);
```